docs/pages/kdaterange.vue

```diff
@@ -77,7 +77,16 @@
 <style lang="css" scoped>
 
   .demo {
+    position: relative;
+    display: flex;
+    align-items: center;
+    justify-content: center;
     height: 850px;
   }
+  @media (max-width: 600px) {
+    .demo {
+      z-index: 9999;
```

Comment:
Sorry for the confusion, I meant can you add this `z-index` property directly within the `.demo` class on line 79 and remove the `@media (max-width: 600px)` portion completely because the KDateRange component is still displayed behind the floating header and side navigation on screens with widths wider than 600px.